server/consumer.go

```diff
@@ -514,7 +514,7 @@ func (mset *stream) addConsumerWithAssignment(config *ConsumerConfig, oname stri
 	if err != nil {
 		mset.mu.Unlock()
 		o.deleteWithoutAdvisory()
-		return nil, fmt.Errorf("error creating store for observable: %v", err)
+		return nil, fmt.Errorf("error creating store for consumer: %v", err)
```

Comment:
btw run into this one often when running the tests with the default limit of file descriptors:

```
=== RUN   TestJetStream_ClusterReconnect/sub_durable/n=3_r=3
    js_test.go:3635: Unexpected error: error creating store for observable: open /var/folders/4h/4z2f4hd12g3fc7v9n7ny5mxc0000gn/T/NODE_0_SUBR3-445106047/jetstream/$G/streams/quux-r3/obs/sd1/meta.inf: too many open files
```